3.0 - Add rights/permissions on roles for reference entities (#279)
akeneo/pim-helpcenter: content/md/permissions-settings/manage-the-interface-and-actions-accesses.md

| Manage asset category permissions | User has access to the `Permissions` tab on the asset category form     |
| Remove an asset category          | User can remove an asset category by clinking on `Delete` in the `...` dropdown                   |

# Rights on reference entities _(EE only)_

| Permission name | Effect on the interface |
|---------------------------|--------------|
| Add an attribute to a reference entity | User can add an attribute to a reference entity in `Attributes` tab, using the `Add an attribute` button |
| Delete an attribute for a reference entity | User can delete an attribute of a reference entity in `Attributes` tab, using the `Delete an attribute` button |
| Edit an attribute of a reference entity | User can edit an attribute of a reference entity in `Attributes` tab, using the `edit` icon. |

> | Remove an option    | User can remove an option of a single or multiple options attributes in `Attributes` tab, using the `Manage options` button and the `cross` in the list of options |

Reviewer: ```suggestion
| Remove an option    | User can remove an option of a single or multiple options attribute in the `Attributes` tab, using the `Manage options` button and the `cross` in the list of options |
```

> | Edit options | User can manage the options for a single or multiple options attributes in `Attributes` tab, using the `Manage options` button |

Reviewer: ```suggestion
| Edit options | User can manage the options for a single or multiple options attribute in the `Attributes` tab, using the `Manage options` button |
```

> | Create a record | User can add a record to a reference entity in `Records` tab, using the `Add new` button  |

Reviewer: ```suggestion
| Create a record | User can add a record to a reference entity in the `Records` tab, using the `Add new` button  |
```

> | Delete a record | User can delete a record in `Records` tab, using the `deletion icon` in the grid or `...` for other actions and then `Delete` in the record form |

Reviewer: ```suggestion
| Delete a record | User can delete a record in the `Records` tab, using the deletion icon in the grid or click on `...` for other actions and then on `Delete` in the record form |
```

> | Delete all entity records | User can delete all records for a reference entity in `Records` tab, using `...` for other actions and then `Delete all records` |

Reviewer: ```suggestion
| Delete all entity records | User can delete all the records of a reference entity in the `Records` tab, clicking on `...` for other actions and then on `Delete all records` |
```

> | Edit the records | User can edit a record of a reference entity in `Records` tab, using the `Edit` icon in the grid or by double clicking on the record |

Reviewer: ```suggestion
| Edit the records | User can edit a record of a reference entity in the `Records` tab, using the `Edit` icon in the grid or by double-clicking on the record |
```

> | Create an entity | User can create a new reference entity in `Entities` menu, using the `Create` button |

Reviewer: ```suggestion
| Create an entity | User can create a new reference entity in the `Entities` menu, using the `Create` button |
```

> | Delete an entity | User can delete a reference entity in `Properties` tab, using `...` for other actions and then `Delete entity` |

Reviewer: ```suggestion
| Delete an entity | User can delete a reference entity in the `Properties` tab, clicking on `...` for other actions and then on `Delete entity` |
```

> | Edit the entity properties | User can edit the properties of a reference entity in `Properties` tab |

Reviewer: ```suggestion
| Edit the entity properties | User can edit the properties of a reference entity in the `Properties` tab |
```

> | Manage reference entity permissions | User can define the permissions for a reference entity in `Permissions` tab |

Reviewer: ```suggestion
| Manage reference entity permissions | User can define the permissions for a reference entity in the `Permissions` tab |
```

> 

Reviewer 2: The descriptions are pretty clear, nevertheless to make them more user-friendly I would use "You can" instead of "User can"

Author: I will keep "User" as it's often set by the administrator, not by the user himself

Reviewer: Agreed with Delph. Also, it is already like this in the other sections of this article.
